src/components/ImageUpload.tsx

```diff
@@ -0,0 +1,66 @@
+import { Paper, Typography } from "@mui/material";
+import { styled } from "@mui/material/styles";
+import { Box } from "@mui/system";
+import { Field, FormikErrors, FormikTouched } from "formik";
+import React from "react";
+import { render } from "react-dom";
+import { useDropzone } from "react-dropzone";
+import { theme } from "./../theme/theme";
+import { StyledPaperInput } from "./StyledComponents";
+
+interface ImageUploadProps {
+  setFieldValue: (field: string, value: any, shouldValidate?: boolean | undefined) => void;
+  errors: FormikErrors<{
+    description: string;
+    genre: string;
+    imageFile: {
+      name: null;
+      size: null;
+      type: null;
+    };
+  }>;
+  touched: FormikTouched<{
+    description: string;
+    genre: string;
+    imageFile: {
+      name: null;
+      size: null;
+      type: null;
+    };
+    releaseDate: string;
+    title: string;
+    yourRole: string;
+  }>;
+}
+
+export const ImageUpload = (props: ImageUploadProps) => {
+  const { setFieldValue, errors, touched } = props;
+  const { getRootProps, getInputProps, isDragActive } = useDropzone({
+    accept: "image/*",
```

Comment:
Is there a way to limit the file size using this library?

We'd also want to protect ourselves from decompression bombs. Not in this PR, but something to keep in mind: https://bomb.codes/